packages/aws-cdk-lib/aws-elasticache/lib/elasticache-cluster.ts

```diff
@@ -0,0 +1,139 @@
+import * as cdk from 'aws-cdk-lib';
+import * as elasticache from 'aws-cdk-lib/aws-elasticache';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { Construct } from 'constructs';
+
+export interface ElastiCacheClusterProps {
+  /**
+   * The VPC where the ElastiCache cluster will be deployed
+   */
+  readonly vpc: ec2.IVpc;
+  
+  /**
+   * The cache engine version to use
+   * @default - 6.x (Redis)
+   */
+  readonly engineVersion?: string;
+  
+  /**
+   * The node type for the ElastiCache cluster
+   * @default - cache.t3.micro
+   */
+  readonly nodeType?: string;
+  
+  /**
+   * Number of cache nodes in the cluster
+   * @default 1
+   */
+  readonly numCacheNodes?: number;
+  
+  /**
+   * The name of the cache parameter group to associate with this cache cluster
+   */
+  readonly parameterGroupFamily?: string;
```

Comment:
it seems it is referring to [this property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticache-parametergroup.html#cfn-elasticache-parametergroup-cacheparametergroupfamily). Since there is a list of allowed values, so it should be of enum type.